rocks-db/src/migrations/spl2022.rs

```diff
@@ -52,7 +52,7 @@ pub struct AssetDynamicDetailsWithoutExtentions {
     pub supply: Option<Updated<u64>>,
     pub seq: Option<Updated<u64>>,
     pub is_burnt: Updated<bool>,
-    pub was_decompressed: Updated<bool>,
```

Comment:
this datatype (AssetDynamicDetailsWithoutExtentions) may be safely removed by the way as all the migrations are turned off at the moment